maps/southern_cross/events/wildlife_encounter.dm

```diff
@@ -0,0 +1,94 @@
+// This event sends random sif mobs at someone in the wilderness, unannounced.
+
+/datum/event2/meta/wildlife_encounter
+	name = "random encounter"
+	departments = list(DEPARTMENT_EVERYONE)
+	chaos = 10
+	event_type = /datum/event2/event/wildlife_encounter
+
+/datum/event2/meta/wildlife_encounter/get_weight()
+	var/explorers = metric.count_people_with_job(/datum/job/explorer) + metric.count_people_with_job(/datum/job/sar)
+
+	if(!explorers)
+		return 0
+	return 20 + explorers * 50
+
+/datum/event2/event/wildlife_encounter
+	var/mob/living/victim = null
+
+/datum/event2/event/wildlife_encounter/set_up()
+	var/list/potential_victims = list()
+	for(var/mob/living/L in player_list)
```

Comment:
I would suggest using `for(var/client/C)` and checking `C.mob` instead of `player_list`, otherwise you could be spawning critters and sending them to kill someone who is SSD.

Reply:
Just noticed the `stat` check, still not sure if `player_list` is the best approach.

Reply:
This is how it's done in the carp event that this one is based off of. I tried looking for an example of how C.mob works but github search had other ideas.

Reply:
Something like this? 
```
	for(var/client/C in GLOB.clients)
		if(!C.mob)
			if(istype(get_turf(C.mob), /turf/simulated/floor/outdoors) && istype(get_area(C.mob),/area/surface/outside/wilderness))
				potential_victims += C.mob
```

Why is this more desirable than getting the mob from the player list? Because the stat check is particularly expensive?

Reply:
Having checked Polariscode it looks like player_list is fine, main reason I was suggesting clients was avoiding logged-out mobs resulting from client getting unexpectedly deleted. If player_list is consistent then yeah you can ignore my comment.

Just for reference `for(var/client/C)` (no `in foo`) is some BYOND sorcery that the compiler optimizes.